src/main/java/io/pravega/perf/ReaderWorker.java

```diff
@@ -47,81 +58,78 @@ public Void call() throws InterruptedException, ExecutionException, IOException
         return null;
     }
 
-    private class EventsReader implements Performance {
-
-        public void benchmark() throws IOException {
-            String ret = null;
-            try {
-                for (int i = 0; i < events; i++) {
-                    final long startTime = System.currentTimeMillis();
-                    ret = readData();
-                    if (ret != null) {
-                        stats.recordTime(startTime, System.currentTimeMillis(), ret.length());
-                    }
+
+    public void EventsReader() throws IOException {
+        String ret = null;
+        try {
+            int i = 0;
```

Comment:
Using a `for` instead of a `while` you can collapse the variable declaration and increment.

Reply:
increment happens on a condition ; hence i changed it to while